content/md/connectivity/connection-dashboard.md

```diff
@@ -7,28 +7,44 @@ ee-only: false
 related: what-is-a-connection, how-to-connect-my-pim, manage-your-connections
 ---
 
-# Monitor your third-party **connections**
+# Overview
 
 In the PIM, there is a dashboard to help you follow your [connections](what-is-a-connection.html) activity. It's called the `Connection dashboard`.
 
 You can access it by clicking on the `Activity` menu, then on the `Connection dashboard` submenu. And here you are! :tada:
 
 ![Connection dashboard](../img/connection-dashboard.png)
 
-From this screen, you can easily follow:
-- the number of products that were **created** by each of your source connections over the last 7 days,
+::: info
+This dashboard is still in its first version. In the future, it will show you many more indicators. Stay tuned! 🙂  
+And also, don't hesitate to fill in the survey at the bottom of the dashboard page to vote for the next KPI you'd like to see in there.
+:::
+
+In this dashboard, for now, you can monitor both your source and destination connections.
+
+# Source connection monitoring
+For your source connections, you can easily follow in this dashboard:
+- the number of products that were **created** by each of them over the last 7 days,
 ![Number of products created](../img/number-of-products-created.png)
-- the number of products that were **created** by all your connections over the last 7 days,
+- the number of products that were **created** by all of them over the last 7 days,
```

Comment:
```suggestion
- the total number of **created** products over the last 7 days,
```

Reply:
I fear it will be less clear that it's the total for all the source connections.

Reply:
```suggestion
- the number of products that were **created** over the last 7 days by all your sources,
```